radis/__init__.py

```diff
@@ -98,7 +98,13 @@
 
 
 # %% Global constants
-from .params import AUTO_UPDATE_SPEC, DEBUG_MODE, OLDEST_COMPATIBLE_VERSION, USE_CYTHON
+from .params import (
+    AUTO_UPDATE_SPEC,
+    DEBUG_MODE,
+    OLDEST_COMPATIBLE_VERSION,
+    USE_CYTHON,
+    WARN_THRESHOLD,
```

Comment:
to keep things together you could also keep the ERROR_THRESHOLD here. And now that they're mixed with other parameters let's call them more explicitely. What do you think of :
```
GRIDPOINTS_PER_LINEWIDTH_WARN_THRESHOLD
GRIDPOINTS_PER_LINEWIDTH_ERROR_THRESHOLD
```
(long, but explicit)